ssc/cmod_utilityrate5.cpp

```diff
@@ -586,6 +596,22 @@ class cm_utilityrate5 : public compute_module
             monthly_two_meter_sales(12),
             monthly_peak_wo_sys(12), monthly_peak_w_sys(12), // can't re-use these due to their role in billing demand
             monthly_true_up_credits(12); // Realistically only one true-up month will be non-zero, but track them all for monthly outputs
+        std::vector<ssc_number_t> monthly_revenue_w_sys_batt(12), monthly_revenue_wo_sys_batt(12),
+            monthly_fixed_charges_batt(12), monthly_minimum_charges_batt(12),
+            monthly_ec_charges_batt(12),
+            monthly_ec_charges_gross_batt(12),
+            monthly_nm_dollars_applied_batt(12),
+            monthly_excess_dollars_earned_batt(12),
+            monthly_excess_kwhs_earned_batt(12),
+            monthly_net_billing_credits_batt(12),
+            monthly_ec_rates_batt(12),
+            monthly_salespurchases_batt(12),
+            monthly_load_batt(12), monthly_system_generation_batt(12), monthly_elec_to_grid_batt(12),
+            monthly_elec_needed_from_grid_batt(12),
+            monthly_cumulative_excess_energy_batt(12), monthly_cumulative_excess_dollars_batt(12), monthly_bill_batt(12), monthly_peak_batt(12), monthly_test_batt(12),
+            monthly_two_meter_sales_batt(12),
+            monthly_true_up_credits_batt(12); // Realistically only one true-up month will be non-zero, but track them all for monthly outputs
```

Comment:
I'm confused by these new inputs. It looks like they aren't actually used?

Additionally a lot of the computations inside ur_calc don't seem to make it outside of that function. 

It looks like the only thing that is happening here is the computation of e_batt_cy and p_batt_cy, is the rest of the code needed?

Reply:
Originally I thought there would need to be changes to utility rate code to calculate the grid charging costs. After conversations I ended up not needing much or any of these code changes. I tried to remove changes but I see I missed a lot. I'm looking through it but I'm fairly confident this can be reverted to develop